recipes-core/systemd/systemd_%.bbappend

```diff
@@ -3,6 +3,10 @@ quotaon_srv = "quotaon.service"
 
 FILESEXTRAPATHS_prepend := "${THISDIR}/${PN}:"
 
+PACKAGECONFIG[p11kit] = "-Dp11kit=true -Dhomed=false, -Dp11kit=false, p11-kit"
```

Comment:
-Dp11kit=true, -Dp11kit=false is this correct options?

Reply:
yes, it is correct  plese see prameter description from yoicto manual
 PACKAGECONFIG[f1] = "--with-f1,--without-f1,build-deps-f1,rt-deps-f1"